CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.22.1 / 2020-11-03
+
+* [BUGFIX] Fix potential "mmap: invalid argument" errors. #8061
```

Comment:
This is a bit vague, can we describe what logic was fixed?

Reply:
Added something more elaborate, let me know if that's better.

Reply:
I think it'd be good to mention that this is about the head chunks.

Reply:
Added. Let me know how that is.